roles/sap_ha_set_hana/tasks/cluster_constraint.yml

```diff
@@ -3,25 +3,32 @@
 
 - name: "SAP Pacemaker Hana - check constraint order HANA TOPOLOGY"
   ansible.builtin.shell: |
```

Comment:
`.shell` used when others are converted to `.command` ?

Reply:
Yes, because of the pipes.